src/matter/cluster/server/OperationalCredentialsServer.ts

```diff
@@ -21,7 +21,7 @@ import { tryCatch } from "../../../error/TryCatchHandler";
 import { FabricNotFoundError } from "../../fabric/FabricManager";
 import { Logger } from "../../../log/Logger";
 
-const logger = Logger.get("OperationalCredentialsServer");
+const logger = Logger.get("OperationalCredentialClusterServer");
```

Comment:
This doesn't match the file name. If the file name is incorrect, it should also be updated.

Reply:
renamed